src/Controller/PeopleController.php

```diff
@@ -0,0 +1,137 @@
+     *     "/validate/{element}",
+     *     name="validatePerson",
+     *     requirements={"POST"}
+     * )
+     */
+    public function validate(Request $request, string $element)
+    {
+        try {
+            $input = json_decode($request->getContent(), true)['input'];
+        } catch (Exception $e) {
+            return new JsonResponse(['error' => 'Invalid method'], Response::HTTP_BAD_REQUEST);
+        }
+
+        $students = $this->getStudents();
+        switch ($element) {
+            case 'name':
+                return new JsonResponse(['valid' => in_array(strtolower($input), $students)]);
+        }
+
+        $projects = $this->getProjects();
+        switch ($element) {
+            case 'project':
+                return new JsonResponse(['valid' => in_array(strtolower($input), $projects)]);
+        }
+
+        return new JsonResponse(['error' => 'Invalid method'], Response::HTTP_BAD_REQUEST);
+    }
+
+    private function getStorage()
+    {
+        return /** @lang json */
+        '{
+          "nedarykpats": {
+            "name": "Nedaryk pats",
+            "mentors": [
+              "Laurynas"
+            ],
+            "students": [
+              "Martyna",
+              "Aurimas",
+              "Vilius"
+            ]
+          },
+          "savanoryste": {
+            "name": "Car booking",
+            "mentors": [
+              "Tomas"
+            ],
+            "students": [
+              "Ignas",
+              "Dovydas",
+              "Darius"
+            ]
+          },
+          "curlybrackets": {
+            "name": "Maisto dalinimosi sistema",
+            "mentors": [
+              "Paulius"
+            ],
+            "students": [
+              "Roman",
+              "Marijus",
+              "Angelika"
+            ]
+          },
+          "hobby": {
+            "name": "Hobby",
+            "mentors": [
+              "Ieva"
+            ],
+            "students": [
+              "Miroslav",
+              "Viktoras",
+              "Lukas"
+            ]
+          },
+          "hack<b>er</b>\'is po .mySubdomain &project=123": {
+            "name": "\' OR 1 -- DROP DATABASE",
+            "mentors": [
+              "<b>Ponas</b> Programišius"
+            ],
+            "students": [
+              "Aurelijus Banelis",
+              "<b>Ir</b> jo \"geras\" draug\'as"
+            ]
+          }
+        }';
+    }
+
+    private function getStudents(): array
+    {
+        $students = [];
+        $storage = json_decode($this->getStorage(), true);
+        foreach ($storage as $teamData) {
+            foreach ($teamData['students'] as $student) {
+                $students[] = strtolower($student);
+            }
+        }
+        return $students;
+    }
+
+    private function getProjects(): array
+    {
+        $projects = [];
+        $storage = json_decode($this->getStorage(), true);
+        foreach ($storage as $teamDataSecond => $value) {
```

Comment:
Dabar veikia, bet gal praverst požiūrio praplėtimui.
```suggestion
        foreach (array_keys($storage) as $teamDataSecond) {
```
Su `array_keys` funkcija iš karto gaunami visos `keys` reikšmės, tada nebereikia naudoti `$t` kintamojo, kuris jokios naudos neduoda.

Plačiau:
 * https://www.php.net/manual/en/function.array-keys.php